src/react-components/profile-entry-panel.js

```diff
@@ -44,7 +46,14 @@ class ProfileEntryPanel extends Component {
         avatarId: this.state.avatarId
       }
     });
+
     this.props.finished();
+    this.props.history.goBack();
+
+    // We may need to go to a new path after saving.
+    if (this.props.location.state.postPushPath) {
+      this.props.history.push(this.props.location.state.postPushPath);
```

Comment:
Hmm.. This goBack + conditional push feels odd. Being able to hit back to close the modal is maybe nice but I am not sure I expect to be able to go back to re-open a modal.